apm-server/README.md

```diff
@@ -8,8 +8,8 @@ The design and code is less mature than official GA features and is being
 provided as-is with no warranties. Alpha features are not subject to the support
 SLA of official GA features (see [supported configurations][] for more details).
 
-**Warning**: This branch is used for development, please use [7.9.2][https://github.com/elastic/helm-charts/releases/tag/7.9.2] release for released version.
-
+**Warning**: This branch is used for development, please use the latest [7.9](https://github.com/elastic/helm-charts/releases)
```

Comment:
Another comment, maybe we could mention 7.x instead 7.9 see https://github.com/elastic/helm-charts/pull/860#discussion_r509331124